src/lib/index.jsx

```diff
@@ -300,7 +307,12 @@ class OtpInput extends Component<Props, State> {
           focus={activeInput === i}
           value={otp && otp[i]}
           onChange={this.handleOnChange}
-          onKeyDown={this.handleOnKeyDown}
+          onKeyDown={(e: Object) => {
+            if (onKeyDown.keyCode === e.keyCode)
```

Comment:
Instead of doing this check, you could make the `onKeyDown` just be a function.
Then do something like:

```
if (this.props.onKeyDown) {
  this.props.onKeyDown(event)
  if (event.isDefaultPrevented()) {
    return
  } 
}
this.handleOnKeyDown(event)
```